lib/pdf_attachment_reporter.rb

```diff
@@ -31,13 +32,18 @@ def initialize(opts = {})
   def pdfs_by_organisation
     second_time_period_date = @last_time_period_days.days.ago.to_date
 
-    live_organisation_names = Organisation.where(govuk_status: 'live').map(&:name) << POLICY_GROUPS
+    live_organisation_names = Organisation.where(govuk_status: 'live').includes(:translations).map(&:name) << POLICY_GROUPS
 
     live_organisation_published_pdfs_total_counts_hash = Hash[live_organisation_names.map { |o| [o, 0] }]
     live_organisation_published_pdfs_since_first_period_counts_hash = Hash[live_organisation_names.map { |o| [o, 0] }]
     live_organisation_published_pdfs_since_second_period_counts_hash = Hash[live_organisation_names.map { |o| [o, 0] }]
 
-    unique_published_pdf_attachments.each do |attachment|
+    unique_pdf_attachments = unique_published_pdf_attachments
```

Comment:
I'm not sure what the purpose of this assignment is? The method only returns an `ActiveRecord::Relation` object, so no queries will run until a "kicker" method is called, such as `each` or `count`.